e2e-tests/functions

```diff
@@ -498,6 +498,7 @@ deploy_version_service() {
 		--from-file "${TESTS_CONFIG_DIR}/operator.9.9.9.ps-operator.json"
 
 	kubectl apply -n "${NAMESPACE}" -f "${TESTS_CONFIG_DIR}/vs.yaml"
+	sleep 10
```

Comment:
Can we just wait until the pods are ready instead of adding sleep?

Reply:
it was copypast from PG but I think we can :)